src/win32/win_agent.c

```diff
@@ -173,6 +173,11 @@ int local_start()
     }
     minfo("Using notify time: %d and max time to reconnect: %d", agt->notify_time, agt->max_time_reconnect_try);
 
+    if (!Validate_Address(agt->server)){
```

Comment:
Move this piece of code before the notify time operations, to be consistent with the Unix version.